deployment/azuredeploy.json

```diff
@@ -49,6 +49,7 @@
         "staticWebsite": "[toLower(concat('learnclient', replace(variables('uniqueIdentifier'),'-','')))]",
         "keyVaultName": "[toLower(concat('kv-', variables('uniqueIdentifier')))]",
         "StorageAccountSuffix": "[toLower(concat(variables('uniqueIdentifier')))]",
+        "KeyVaultAuditLogsStorageAccountName": "[toLower(concat('auditlogskv' , variables('StorageAccountSuffix')))]",

```

Comment:
> What I meant from my previous comment was something like below. This should group KV resources together on the UI as well. Let me know if this seems better: -
```suggestion json
        "KeyVaultAuditLogsStorageAccountName": "[toLower(concat(variables('keyVaultName'), '-auditlogs' ))]",
```

Reply:
storage account names cannot include special characters, so basically the storage account suffix is the unique identifier without the `-`.
Right now the keyvault name is `kv-<storageAccountSuffix>` and according to this code, the storage account name for logging is `auditlogskv<storageAccountSuffix>`

Having said that, i'll change it to `kv<storageAccountSuffix>auditlogs`